pkg/channel/fanout/fanout_message_handler.go

```diff
@@ -211,3 +281,22 @@ func (f *MessageHandler) makeFanoutRequest(ctx context.Context, message binding.
 		sub.RetryConfig,
 	)
 }
+
+type dispatchResult struct {
+	err  error
+	info channel.DispatchExecutionInfo
+}
+
+type TypeExtractorTransformer string
```

Comment:
This should live in `kncloudevents` IMO

Reply:
Ok I will take a look to migrate it there.